recipes/utf8.h/all/conanfile.py

```diff
@@ -0,0 +1,32 @@
+import os
+import re
+from conans import ConanFile, tools
+
+required_conan_version = ">=1.28.0"
```

Comment:
Is this needed? I believe it was added for components names or similar

Reply:
I think it's more or less informational only. CCI policy is to require latest client anyway.

Reply:
Indeed, it can be removed.

Reply:
Not needed, sorry. It was a leftover (copy/paste) from another recipe with `validate` method.